packages/calypso-products/src/main.ts

```diff
@@ -613,6 +632,8 @@ export function getPlanTermLabel(
 			return translate( 'Annual subscription' );
 		case TERM_BIENNIALLY:
 			return translate( 'Two year subscription' );
+		case TERM_TRIENNIALLY:
+			return translate( 'Three year subscription' );
```

Comment:
I wonder if it's fine to use Three year subscription or should it be a Three-year subscription instead? When I type this, the spellchecker complains about "Three year" and wants me to add "-".

Reply:
I think you're right. I believe "Three-year" is the correct way, I can probably change the biennial one as well.